src/renderer/pages/project/components/script_editor/beat_preview.vue

```
@@ -1,67 +1,105 @@
+    <div v-if="enableMovieGenerate">
+      <!-- Movie preview -->
+      <div class="relative border-2 border-dashed border-gray-300 rounded-lg p-4 text-center">
+        <!-- Generate movie button -->
+        <template v-if="shouldShowGenerateButton && shouldBeGeneratedWithPrompt">
+          <Button
+            variant="ghost"
+            size="icon"
+            class="absolute -left-3 -top-3 z-10 bg-white border border-gray-300 rounded-full shadow hover:bg-gray-100 transition-colors w-8 h-8 flex items-center justify-center"
+            @click="generateMovie"
+            :disabled="!enableMovieGenerate || isMovieGenerating || props.toggleTypeMode"
+            :title="props.toggleTypeMode ? 'Change beat type first' : isMovieGenerating ? 'Generating...' : 'Generate movie'"
```

ESLint

[error] Replace `props.toggleTypeMode·?·'Change·beat·type·first'·:·isMovieGenerating·?·'Generating...'·:·'Generate·movie'` with `⏎··············props.toggleTypeMode·?·'Change·beat·type·first'·:·isMovieGenerating·?·'Generating...'·:·'Generate·movie'⏎············`

(prettier/prettier)